submissions/EmmanuelUdoh/Database/my database task.txt

```diff
@@ -1 +1,3 @@
-﻿https://docs.google.com/document/d/1CYheWrOSiOr4iTLEjcLU7QQPFLDwg0tweMKnyvvaWoA/edit?usp=sharing
```

Comment:
Hello @Eudoh940 You would typically want your database submission to be in a different pull request from your php-intro submission.

Reply:
thanks so much @JohnItoo for this. It is because i am using same branch i used for the database task, that is why the database task.txt is still appearing. morover the task has been merge.